core/src/main/scala/slinky/core/facade/React.scala

```diff
@@ -34,15 +36,30 @@ object ReactElement {
     b.asInstanceOf[ReactElement]
   }
 
-  @inline implicit def optionToElement[T](s: Option[T])(implicit cv: T => ReactElement): ReactElement = {
-    s match {
+  @inline implicit def function0ToElement[E](e: () => E)(implicit cv: E => ReactElement): ReactElement = {
+    cv(e())
+  }
+
+  @inline implicit def iterableToElement[A, B <: Iterable[A]](e: Iterable[A])(implicit cv: A => ReactElement): ReactElement = {
+    e.map(cv).toJSArray.asInstanceOf[ReactElement]
+  }
+
+  @inline implicit def optionToElement[E](o: Option[E])(implicit cv: E => ReactElement): ReactElement = {
+    o match {
       case Some(e) => cv(e)
       case None => null.asInstanceOf[ReactElement]
     }
   }
 
-  @inline implicit def seqElementToElement[T](s: Iterable[T])(implicit cv: T => ReactElement): ReactElement = {
-    s.map(cv).toJSArray.asInstanceOf[ReactElement]
+  @inline implicit def tryToElement[E](t: Try[E])(implicit cv: E => ReactElement): ReactElement = {
```

Comment:
This is also potentially very dangerous. `Try` is used to making dealing with error cases safer by not using exceptions, but this just hides the error in `null` instead of having the user properly handle the error and display it to the user if necessary.

Reply:
Definitely agree. Let's drop this.